.github/workflows/build.yml

```diff
@@ -1,71 +1,113 @@
+        with:
+          push: true
+          tags: '${{ env.docker_repo }}/lumberjack-server:${{ env.docker_tag }}'
+          file: 'scripts/server.dockerfile'
+      - id: 'set_output'
+        run: |-
+          echo "::set-output name=image_server::${{ env.docker_repo }}/lumberjack-server:${{ env.docker_tag }}"
+  build-integration-test-servers:
+    runs-on: 'ubuntu-latest'
+    permissions:
+      contents: 'read'
+      id-token: 'write'
+    needs: 'build-setup'
+    strategy:
+      fail-fast: true
+      matrix:
+        include:
+          - lang: 'go'
+            type: 'grpc'
+            file: './clients/go/test/grpc-app/Dockerfile'
+            run_from: './'
+          - lang: 'go'
+            type: 'http'
+            file: './clients/go/test/shell/Dockerfile'
+            run_from: './'
+          - lang: 'java'
+            type: 'grpc'
+            file: './clients/java-logger/scripts/server_app.dockerfile'
+            run_from: './'
+          - lang: 'java'
+            type: 'http'
+            file: './clients/java-logger/scripts/shell_app.dockerfile'
+            run_from: './'
+    steps:
+      - name: 'Configure environment variables'
+        if: ${{ inputs.env-context != '{}' }}
+        run: |-
+          jq -r <<< '${{ inputs.env-context }}' | jq -r "to_entries | map(\"\(.key)=\(.value|tostring)\") | .[]" >> $GITHUB_ENV
+      - name: 'Checkout'
+        uses: 'actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b' # ratchet:actions/checkout@v3
       - id: 'auth'
         name: 'Authenticate to Google Cloud'
         uses: 'google-github-actions/auth@ceee102ec2387dd9e844e01b530ccd4ec87ce955' # ratchet:google-github-actions/auth@v0
         with:
-          workload_identity_provider: 'projects/${{ env.PROJECT_NUMBER }}/locations/global/workloadIdentityPools/github-pool/providers/github-provider'
-          service_account: 'gh-access-sa@lumberjack-dev-infra.iam.gserviceaccount.com'
+          workload_identity_provider: '${{ env.wif_provider }}'
+          service_account: '${{ env.wif_service_account }}'
           token_format: 'access_token'
-
       - name: 'Authenticate to Artifact Registry'
         uses: 'docker/login-action@49ed152c8eca782a232dede0303416e8f356c37b' # ratchet:docker/login-action@v2
         with:
           username: 'oauth2accesstoken'
           password: '${{ steps.auth.outputs.access_token }}'
-          registry: '${{ env.REGISTRY }}'
-
-      - name: 'Build and push container'
+          registry: '${{ env.docker_registry }}'
+      - name: 'Build the integration test server container and push to the registry'
+        uses: 'docker/build-push-action@c84f38281176d4c9cdb1626ffafcd6b3911b5d94' # ratchet:docker/build-push-action@v3
+        with:
+          push: true
+          tags: '${{ env.docker_repo }}/ljc-${{matrix.lang}}-${{ matrix.type }}:${{ env.docker_tag }}'
```

Comment:
I was abbreviating lumberjack-client. I'm using that image name as part of the cloud run revision name and I needed to shave characters to get below the ~60 character maximum. I can move it to a variable and throw a comment on it or use something else here but it will need to be short.